packages/website/components/meta-context.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+interface ContextInterface {
+  siteName: string;
+}
+
+const ContextMeta = React.createContext<ContextInterface>({ siteName: '' });
+
+export default ContextMeta;
+
+/* eslint-disable import/no-unresolved */
+// @ts-ignore
```

Comment:
does @ts-ignore have a way to disable specific errors in the same way eslint does?

Reply:
Nope. It's a type error not a specific rule.